src/Utilities/Clock.h

```diff
@@ -19,9 +19,33 @@
 
 #include <sys/time.h>
 #include <stddef.h>
+#include <chrono>
 
 namespace qmcplusplus
 {
+
+
+// Implements a std::chrono clock
+// See https://github.com/korfuri/fake_clock
+
+class FakeChronoClock
```

Comment:
Can we use FakeChronoClock to replace all FakeCPUClock?

Reply:
The runtime manager was not switched in this PR, but could be changed in the future (and the the FakeCPUClock removed)
I would prefer not to put that change in this PR, in order to keep PR's smaller.

Reply:
Hopefully the switch will happen relatively soon.

Reply:
Indeed.